cmd/podman/save.go

```diff
@@ -77,6 +77,9 @@ func saveCmd(c *cli.Context) error {
 	if !c.Bool("quiet") {
 		writer = os.Stdout
 	}
+	if !c.IsSet("output") {
+		writer = nil
```

Comment:
Could we do os.Stderr to get the status bar still, but not have it redirected to the file?

Reply:
fixed.